main.py

```diff
@@ -26,57 +26,19 @@ def MD(options):
 
     # Use Asap for a huge performance increase if it is installed
     use_asap = options["use_asap"]
-
-    # TODO: Make some of these optional by creating factory function for
-    # LennardJones
-    atomic_number = options["atomic_number"]
-    epsilon = options["epsilon"] * units.eV
-    sigma = options["sigma"] * units.Ang
-    cutoff = options["cutoff"] * units.Ang
     iterations = options["iterations"] if options["iterations"] else 200
     interval = options["interval"] if options["interval"] else 10
 
     if use_asap:
         print("Running with asap")
-        from asap3 import EMT
         from asap3.md.verlet import VelocityVerlet
-        from asap3 import LennardJones
```

Comment:
Looks like we need LennardJones pot as a good default if openKIM doesn't work. Maybe this should be added back now?